README.md

```diff
@@ -115,6 +122,31 @@ file, they can be found in [`scripts`](./docker/package/scripts) and
 [`defaults`](./docker/package/defaults) folders respectively.
 
 
+### Multiple similar systemd services
+
+It's possible to run multiple same services, e.g. two `tezos-node`s that run different
+networks.
+
+In order to do that you should copy systemd service file and config file, e.g.:
+```
+cp /lib/systemd/system/tezos-node.service /lib/systemd/system/tezos-node-zeronet.service
+cp /etc/default/tezos-node /etc/default/tezos-node-zeronet
```

Comment:
Let's provide `tezos-node-mainnet`, `tezos-node-carthagenet` and `tezos-node-delphinet` services out of the box. Ideally without code duplication. It's a bit inconvenient to do these commands manually.

Reply:
Updated. I've changed packages and services logic creation a bit, please check